Class/Practise 3.2/src/Color.cpp

```diff
@@ -0,0 +1,20 @@
+#include "Color.hpp"
+#include <cmath>
+
+Color::Color(int r, int g, int b) : r(r), g(g), b(b) {}
```

Comment:
I don't see input validation here - we should make some decision for such cases as you did a decision to clamp the values to interval in case of magic color generation